tests/integration/src/test/scala/dcos/metronome/integrationtest/utils/MetronomeFramework.scala

```diff
@@ -142,6 +142,9 @@ object MetronomeFramework {
         }
     }(collection.breakOut)
 
+    lazy val url = s"http://localhost:$httpPort"
+    lazy val facade = new MetronomeFacade(url)
```

Comment:
where are those used?

Reply:
See line 40 in tests/integration/src/test/scala/dcos/metronome/integrationtest/MetronomeITBase.scala.